lib/graphlient/errors/graphql_error.rb

```diff
@@ -1,7 +1,7 @@
 module Graphlient
   module Errors
     class GraphQLError < Error
-      attr_reader :responsee
+      attr_reader :response
```

Comment:
While this is a just typo, this is technically a breaking change and could break existing users and it should ho through the deprecation cycle.

Reply:
good point. is there a deprecation guide? i know i should add back the old attr_reader but is there any particular way i should mark this for deprecation?

Reply:
I believe we follow [semver](https://semver.org/) in terms of versioning. Then the old method could output a warning when called:

```ruby
class GraphQLError < Error
  attr_reader :response

  ...

  def responsee
    warn "The `#responsee' method is deprecated since it has a typo. Please use the `#response' method instead."

    response
  end
end
```

Reply:
added a deprecation notice. i returned `@responsee` though (always nil) since that was the old behavior.

Reply:
Oh I do not notice that it always returns nil. That means it is less likely that people actually are using it and we do not have to worry about compatibility. I am sorry for the back and forth, but would you just remove it? Renaming should just be fine.

Reply:
@yuki24 we can leave it deprecated for now since @jmondo has already did deprecation, i can remove it later.